infokala/models.py

```diff
@@ -1,17 +1,18 @@
 # encoding: utf-8
+from __future__ import unicode_literals
 
 from warnings import warn
 
+from dateutil.tz import tzlocal
```

Comment:
pystyyks isortia konffaan tän osalta?

ei ny mikään musthave mut näitä samalla 5-jakoisella rytmillä laulettuja projekteja on kasa muitakin